examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -15,7 +15,6 @@
    "source": [
```

Comment:
<p>Imports <code>DatasetIndex</code> and <code>W</code>  are not used in this notebook.</p><p> </p><p>Also, why remove <code>warnings.filterwarnings("ignore")</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>